app/src/main/java/org/mozilla/fenix/library/recentlyclosed/RecentlyClosedFragmentStore.kt

```diff
@@ -41,5 +45,6 @@ private fun recentlyClosedStateReducer(
 ): RecentlyClosedFragmentState {
     return when (action) {
         is RecentlyClosedFragmentAction.Change -> state.copy(items = action.list)
+        is RecentlyClosedFragmentAction.ChangeSelection -> state.copy(selectedTabs = action.tabs)
```

Comment:
Like the other implementations of the selection holder, let's add the `Select/Deselect` logic to our store instead of the controller. Consistency should make it easier to refactor if we decide to update the way this logic works.

Have a look at either the Bookmarks/History/Downloads/TabsTray stores for how multi-select was implemented.